lib/ansible/modules/monitoring/zabbix/zabbix_template.py

```diff
@@ -605,6 +605,10 @@ def import_template(self, template_content, template_type='json'):
             if LooseVersion(api_version).version[:2] <= LooseVersion('3.2').version:
                 update_rules['applications']['updateExisting'] = True
 
+            if (LooseVersion(api_version).version[:3] >= LooseVersion('4.0.16').version) or \
```

Comment:
With this condition expression, the following version will be matched and an error will occur.

* 4.4.0
* 4.4.1
* 4.4.2
* 4.4.3

There must be conditions expression to exclude the above versions.

Reply:
Hmm, what about using `4.1` instead of `4.0.17`? I think it is really improbable that 4.0 branch will be getting that much updates.

Reply:
Hm... I see. I think best would be to check if it's in the 4.0 branch first and then if it's >= .16.
the >= 4.4.4 should be sufficient anyways.